packages/auth/src/authClients/supertokens.ts

```diff
@@ -0,0 +1,45 @@
+import Sessions from "supertokens-auth-react/recipe/session";
+
+import type { AuthClient } from './';
+
+export interface SuperTokensUser {
+  userId: string;
+  accessTokenPayload: any;
+}
+
+export type SuperTokens = AuthClient;
+
+export const supertokens = (client: {
+  authRecipe: any,
+}): AuthClient => {
+  return {
+    type: "supertokens",
+    client: undefined,
+    login: async () => client.authRecipe.redirectToAuth('signin'),
+
+    signup: async () => client.authRecipe.redirectToAuth('signup'),
+
+    logout: async () => Sessions.signOut(),
+
+    getToken: async (): Promise<string | null> => {
+      if (await Sessions.doesSessionExist()) {
+        let accessTokenPayload = await Sessions.getAccessTokenPayloadSecurely();
+        let jwtPropertyName = accessTokenPayload["_jwtPName"];
+        return accessTokenPayload[jwtPropertyName];
+      } else {
+        return null;
```

Comment:
Also per RW code style, no semicolons please. Thanks!

You may want to run `yarn lint` to avoid lint warnings in CI as well.